platform/ios/src/MGLMapView.mm

```diff
@@ -1015,6 +1033,12 @@ - (void)setContentInset:(UIEdgeInsets)contentInset animated:(BOOL)animated
 
 - (void)setContentInset:(UIEdgeInsets)contentInset animated:(BOOL)animated completionHandler:(nullable void (^)(void))completion
 {
+    // makes sure the insets don't have negative values that could hide the ornaments
+    // thus violating our ToS
+    contentInset = UIEdgeInsetsMake(fmaxf(contentInset.top, 0),
```

Comment:
this is diverging from iOS contentInset - negative values are allowed in contentInset. How about fixing ornaments positioning (clamping position to visible content area) somewhere else or documenting the change here, too.